README.md

```diff
@@ -524,7 +524,10 @@ pycm.ConfusionMatrix(classes: ['L1', 'L2', 'L3'])
 
 ### Parameter recommender
 
-This option has been added in `version 1.9` in order to recommend most related parameters considering the characteristics of the input dataset. The characteristics according to which the parameters are suggested are balance/imbalance and binary/multiclass. All suggestions can be categorized into three main groups: imbalanced dataset, binary classification for a balanced dataset, and multi-class classification for a balanced dataset. The recommendation lists have been gathered according to the respective paper of each parameter and the capabilities which had been claimed by the paper.
+This option has been added in `version 1.9` to recommend the most related parameters considering the characteristics of the input dataset. 
+According to the parameters' suggestion, the characteristics are as follows: balance/imbalance and binary/multi-class.
```

Comment:
This sentence is ambiguous. I suggest to change it as follows. 

The suggested parameters are selected according to some characteristics of the input such as being balance/imbalance and binary/multi-class.